### PR TITLE
Updated Cloud FAQ for IP allowlists.

### PR DESCRIPTION
**Pull request by russjones**
This question often comes up in two contexts.

> Do we support configuring Teleport Cloud to only allow connections from a specific IP-range?

The answer is no. We encourage the use of client certificates and device trust as they provide stronger security properties.

> Can we share the IP addresses that Teleport Cloud is available at so users can configure their own firewalls to allow traffic to Teleport Cloud?

The answer for this is also no. While this [may be possible with AWS GA](https://docs.aws.amazon.com/global-accelerator/latest/dg/about-accelerators.eip-accelerator.html) we don't have plans for it at the moment and offered an alternative (stable DNS name and TLS).

### CODE REVIEW

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 1]
```diff
@@ -173,12 +173,21 @@ for customer data including session recordings and user records.
...
+utilized to verify the integrity of any outbound connections to Teleport
+Enterprise Cloud.
 
-No. We believe mTLS with strong identity is the best standard available for authenticating and access clients.
-We are also investigating additional and new features that can be used as a defense in depth strategy for use with Teleport Enterprise Cloud in 2022.
+## Can I configure an allowlist of inbound IP addresses to Teleport Enterprise Cloud?
+
+We do not have plans to offer support for inbound connection IP allowlists.
+
+We believe mTLS with strong user and [device
```
Comment (klizhentas) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:188]: this should be inline link url `[device identity](../../access-controls/guides/device-trust.mdx)`

**Comment by jimbishopp** [comment]
We do have static IP addresses configured for our GAs. However, if we swap GAs like we did for the EKS migration, or make other changes to the GAs such as our upcoming planning change to disable TCP termination, the IP addresses will change.

**Comment by russjones** [comment]
@jimbishopp Okay thanks, I think it's better not to offer this to customers then. It will cause more trouble than the value returned.

**Comment by sclevine** [comment]
If customers need this feature for security/compliance reasons, we could do what Cloudflare does: https://www.cloudflare.com/ips/

**Review by klizhentas — APPROVED** [review]

**Comment by russjones** [comment]
@sclevine Good to know the option exists, but lets hold on that until our infrastructure stabilizes some more.

**Review by zmb3 — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 2]
```diff
@@ -173,12 +173,29 @@ for customer data including session recordings and user records.
...
 
-## Can I configure Teleport Enterprise Cloud to only allow connections from an allowlist of IP addresses?
+Teleport Enterprise Cloud infrastructure is elastic and IP addresses can and do
+change when we make infrastructure changes. This would make the maintenance of
+IP allowlists a cumbersome and brittle process for customers.
 
-No. We believe mTLS with strong identity is the best standard available for authenticating and access clients.
-We are also investigating additional and new features that can be used as a defense in depth strategy for use with Teleport Enterprise Cloud in 2022.
+However, we do provide customers with a stable tenant DNS name backed with a
+TLS certificates from [letsencrypt.org](https://letsencrypt.org) that can be
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:184]: I know this is already merged, but you're mixing singular and plural with "backed with a TLS certificates".

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 3]
```diff
@@ -173,12 +173,29 @@ for customer data including session recordings and user records.
...
+utilized to verify the integrity of any outbound connections to Teleport
+Enterprise Cloud.
+
+## Can I configure an allowlist of inbound IP addresses to Teleport Enterprise Cloud?
+
+We do not have plans to offer support for inbound connection IP allowlists.
+
+We believe mTLS with strong user and [device
+identity](../../access-controls/guides/device-trust.mdx) provide the best
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:193]: As above, because "mTLS" is singular, it should be "provides".

**Pull request merged**